pkgs/by-name/ta/tabby-terminal/fix-app-version.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/app/package.json b/app/package.json
```

Comment:
Honestly instead of patching package.json, I'd either just `substituteInPlace` or use `jq` to achieve a more elegant fix

```
substituteInPlace app/package.json \
  --replace-fail "1.0.0-alpha.1" "${version}"

# or this (needs jq & moreutils)
jq '.version = "${version}"' app/package.json | sponge app/package.json
```

Reply:
Can't you just use the `>` operator as in:
`jq '.version = "${version}"' app/package.json > app/package.json`

Reply:
Nope, since jq doesn't natively support in-place editing and the file might get corrupted